pytest.ini

```diff
@@ -1,3 +1,4 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = testsuite.settings
+log_cli = true
```

Comment:
I'm not a fan of this setting. pytest already captures logs and prints them for failed tests. Let's find another way to fix your problem.